### ISSUE TITLE
Installation Failed - Fedora 32 - Python 3.8.2

### ISSUE THREAD

**Issue by pazooki** [InsightSoftwareConsortium/ITKIOMeshSTL#34]
```
(venv) m@localhost> python -m pip install itk-iomeshstl
ERROR: Could not find a version that satisfies the requirement itk-iomeshstl (from versions: none)
ERROR: No matching distribution found for itk-iomeshstl
(venv) m@localhost> uname -a
Linux localhost.localdomain 5.6.8-300.fc32.x86_64 #1 SMP Wed Apr 29 19:01:34 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
# Fedora 32
# Python 3.8.2
```
I also tried to build it against the ITK built but ITK built from the scratch also fails and is not an easy build at all on fedora 32 currently...

**Comment by pazooki** [comment]
```
> pip install git+https://github.com/InsightSoftwareConsortium/ITKIOMeshSTL@v1.0.0#egg=itk-iomeshstl
Collecting itk-iomeshstl
  Cloning https://github.com/InsightSoftwareConsortium/ITKIOMeshSTL (to revision v1.0.0) to /tmp/pip-install-l0l3kat5/itk-iomeshstl
  Running command git clone -q https://github.com/InsightSoftwareConsortium/ITKIOMeshSTL /tmp/pip-install-l0l3kat5/itk-iomeshstl
  Running command git checkout -q 24bd405e7dfb3c30213720ff750375c6eb3fadb0
    ERROR: Command errored out with exit status 1:
     command: /home/mehrdad/PycharmProjects/BeuNose/venv/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-l0l3kat5/itk-iomeshstl/setup.py'"'"'; __file__='"'"'/tmp/pip-install-l0l3kat5/itk-iomeshstl/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-e537qd3r
         cwd: /tmp/pip-install-l0l3kat5/itk-iomeshstl/
    Complete output (8 lines):
    running egg_info
    creating /tmp/pip-pip-egg-info-e537qd3r/itk_iomeshstl.egg-info
    writing /tmp/pip-pip-egg-info-e537qd3r/itk_iomeshstl.egg-info/PKG-INFO
    writing dependency_links to /tmp/pip-pip-egg-info-e537qd3r/itk_iomeshstl.egg-info/dependency_links.txt
    writing requirements to /tmp/pip-pip-egg-info-e537qd3r/itk_iomeshstl.egg-info/requires.txt
    writing top-level names to /tmp/pip-pip-egg-info-e537qd3r/itk_iomeshstl.egg-info/top_level.txt
    writing manifest file '/tmp/pip-pip-egg-info-e537qd3r/itk_iomeshstl.egg-info/SOURCES.txt'
    error: package directory 'itk' does not exist
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by thewtex** [comment]
Hello @pazooki , version 2.1.0, just released in avaliable for Python 3.8.

```
python -m pip install itk-iomeshstl
```

Enjoy ITK!
